### PR TITLE
test(client): add regression tests for mongo raw iTX issue

### LINKED ISSUE #15532: Add tests for mongodb's `findRaw`, `aggregateRaw` and `runCommandRaw` in sequential transactions
Creating this issue as a reminder that we are missing functional tests for `findRaw`, `aggregateRaw` and `runCommandRaw` in sequential transactions, we currently have no guarantees that it's working as expected.

+Reminder: Check if this is tested on the engines

### PR DESCRIPTION
**Pull request by aqrln**
Add regression tests for using `findRaw`, `aggregateRaw` and `runCommandRaw` inside interactive transactions with MongoDB.

Ref: https://github.com/prisma/prisma/issues/13405
Ref: https://github.com/prisma/prisma/issues/14543
Ref: https://github.com/prisma/prisma-engines/pull/3206
Closes: https://github.com/prisma/prisma/issues/15532

### CODE REVIEW

File: packages/client/package.json [hunk 1]
```diff
@@ -147,7 +147,7 @@
     }
   },
   "dependencies": {
-    "@prisma/engines-version": "4.4.0-33.94146edaafca9703aa6ad050adf1b1fd2dff438e"
+    "@prisma/engines-version": "4.4.0-39.integration-query-raw-itx-mongo-a52d215fb2b3cba53b195bbf56b09fc615ddad3d"
```
Comment (janpio) [inline packages/client/package.json:150]: Reset before merging

**Review by millsp — APPROVED** [review]
It looks good! Related question: do you know if we test these on sequential transactions?

**Comment by millsp** [comment]
> It looks good! Related question: do you know if we test these on sequential transactions?

Answer seems to be no, I'll create an issue for it.

**Comment by aqrln** [comment]
> Answer seems to be no, I'll create an issue for it.

I was just going to post an answer to this thread — the answer was no, so I went ahead and added them here, I'll push it in a bit

**Comment by aqrln** [comment]
@millsp done, PTAL

**Comment by aqrln** [comment]
Actually this could've been moved into the the same regression test suite as, thinking about it, sequential transactions were in fact broken too — it's the same piece of logic in the engine where the issue was that is responsible for both. But there's also a benefit in keeping it separate since we get Data Proxy testing for it while the iTX test is skipped.

**Pull request merged**